integration_test.go

```diff
@@ -35,11 +35,13 @@ type putResponse struct {
 func TestIntegration(t *testing.T) {
 	ctx := context.Background()
 	aioImage := "nspccdev/neofs-aio-testcontainer:"
-	versions := []string{"latest"}
+	versions := []string{"0.24.0", "0.25.1", "latest"}
 	key, err := keys.NewPrivateKeyFromHex("1dd37fba80fec4e6a6f13fd708d8dcb3b29def768017052f6c930fa1c5d90bbb")
 	require.NoError(t, err)
 
 	for _, version := range versions {
+		ctx, cancel2 := context.WithCancel(ctx)
```

Comment:
Shading of `ctx` looks not so good in `for` cycle. I propose to use different context name.

```suggestion
		cctx, cancel2 := context.WithCancel(ctx)
```